src/app/map/map.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MapBoxModule } from 'angular-mapbox/module';
+import { MapboxComponent } from './mapbox/mapbox.component';
+
+
+@NgModule({
+  exports: [
+    MapboxComponent
+  ],
+  imports: [
+    CommonModule,
+    MapBoxModule.forRoot(
```

Comment:
This is a small thing, but do we actually need an API key, given we're using our own tiles (other than OpenMapTiles for now)?

Reply:
I think we do, I tried to initialize without an API key and it crashed.  Though, I'm not sure if it was the `angular-mapbox` package I'm using, or if it was `mapboxgl` itself that wouldn't initialize without it.

Reply:
Weird, I just looked at the Mapbox docs and I guess it does, good to know! https://www.mapbox.com/help/how-access-tokens-work/